react-app/src/app/index.js

```diff
@@ -1,36 +1,67 @@
 import React, { Component } from 'react';
-import { Provider } from 'react-redux';
 import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 
-import store from '../redux/store';
-import logo from '../logo.svg';
-
+import { LINKS } from './constants';
 import style from './styles.scss';
+import Topbar from './components/Topbar';
 import RestrictRoute from './components/RestrictRoute';
 import Game from './screens/Game';
 import Login from './screens/Login';
+import User from './screens/User';
+import Help from './screens/Help';
 
 // TODO: react-router-redux implementation
 class App extends Component {
   render() {
+    // TODO: delete div with sessions vars comment
+    // TODO: pass style to Topbar from style object
+    const styleTopbar = {
```

Comment:
Avoid defining styles in javascript and instead define css classes and send them as props